nav/virtual_layers/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(virtual_layers)
+
+find_package(catkin REQUIRED COMPONENTS
+  costmap_2d
+  dynamic_reconfigure
+  roscpp
+  rospy
+  std_msgs
+  tf
+  geometry_msgs #for topic subscription
+  cv_pkg
```

Comment:
We have a dependency on the CV pkg (the one @spencerteetaert made) not cv_pkg (the test pkg) I believe